uber/models.py

```diff
@@ -379,6 +379,82 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
             if not ignore_csrf:
                 check_csrf(params.get('csrf_token'))
 
+    def timespan(self, minute_increment=60):
```

Comment:
Welp, as of [this commit](https://github.com/magfest/ubersystem/commit/e3be9083ad76f1b2923c0b0a8352fb74b8cda4f3)  it now looks like this:
```{{ macros.checkgroup(attendee, 'requested_depts') }}```